codacy/values-crow.yaml

```diff
@@ -0,0 +1,22 @@
+global:
+  codacy:
+    crow:
+      url: <--- "http://codacy.example.com/monitoring" ---> # Crow URL
+crow:
```

Comment:
Does this URL really need to include the path `/monitoring`? Since the same path is also configured on `config.play.http.context` maybe it could be removed from here. My concern is that since the users will edit this value, they may accidentally change or remove the path and something could break.